From 80cc41ca1944d0cd14c4b8585e90a9e4ec952918 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.5-80cc41ca194-20211004T015432Z-avarab@gmail.com>
In-Reply-To: <cover-0.5-00000000000-20211004T015432Z-avarab@gmail.com>
References: <cover-0.5-00000000000-20211004T015432Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 6 Aug 2021 15:10:50 +0200
Subject: [PATCH 1/5] advice.h: don't indent top-level enum
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a style nit introduced in b3b18d16213 (advice: revamp advise API,
2020-03-02) and remove the leading whitespace before "enum".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 advice.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/advice.h b/advice.h
index 601265fd107..694219f5cb2 100644
--- a/advice.h
+++ b/advice.h
@@ -12,7 +12,7 @@ struct string_list;
  * Add the new config variable to Documentation/config/advice.txt.
  * Call advise_if_enabled to print your advice.
  */
- enum advice_type {
+enum advice_type {
 	ADVICE_ADD_EMBEDDED_REPO,
 	ADVICE_ADD_EMPTY_PATHSPEC,
 	ADVICE_ADD_IGNORED_FILE,
-- 
2.33.0.1404.g7bcfc82b295

